__tests__/api.test.js

```diff
@@ -144,3 +144,60 @@ describe("GET /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
+  test("201: Responds with the newly created object in the database", () => {
+    const commentToAdd = {
+      body: "This is just testing text",
+      votes: 1,
+      author: "icellusedkars",
+      article_id: 2,
```

Comment:
Only a username and body should be on the comment to add, the rest should be provided by the server